src/d3a/models/strategy/external_strategies/__init__.py

```diff
@@ -86,11 +86,20 @@ class ExternalMixin:
     def __init__(self, *args, **kwargs):
         self._connected = False
         self.connected = False
+        self.use_template_strategy = False
```

Comment:
Would be better to make this parameter private, only in order to discourage other people from accessing it from outside of this class.